package.json

```diff
@@ -2,6 +2,13 @@
 	"name": "is-symbol",
 	"version": "1.0.2",
 	"description": "Determine if a value is an ES6 Symbol or not.",
+	"files": [
```

Comment:
This is not a fix - "files" is very dangerous. If you want to restrict what's included in the tarball, only `.npmignore` should be used.

Reply:
https://blog.npmjs.org/post/165769683050/publishing-what-you-mean-to-publish

>Whitelisting files with the files array
>
>There’s an even better way of controlling exactly which files are published with your package: whitelisting with the files array. Only 57,000 packages use this method of controlling what goes into them, probably because it requires you to take inventory of your package. **It’s by far the safest way to do it, though.**

Reply:
Just because npm says it, doesn’t make it true. The files array is incredibly dangerous.

The consequence of forgetting to add a file you wish to omit to npmignore is that your package is insignificantly larger for a time.

The consequence of forgetting to add a file you wish to include to files is that your package, and all its consumers, are broken - in a way that’s super difficult to discover and debug from the git repo.

Reply:
Personally, I use a `src` folder so I don't forget.

Reply:
That falls apart with colocated tests, or if you care about ergonomic deep imports, etc, but sure, that’s an approach.

npmignore is still the far safer option.

Reply:
FWIW, I agree with @cmckni3 and use the `files` array in all my open source.